kafka-rest/src/test/java/io/confluent/kafkarest/integration/CustomLogIntegrationTest.java

```diff
@@ -193,6 +195,20 @@ public void setUp(TestInfo testInfo) throws Exception {
       restConfigs.put(KafkaRestConfig.PRODUCE_MAX_REQUESTS_PER_SECOND, 99999);
       restConfigs.put(KafkaRestConfig.PRODUCE_MAX_BYTES_GLOBAL_PER_SECOND, 99999);
       restConfigs.put(KafkaRestConfig.PRODUCE_MAX_BYTES_PER_SECOND, 1);
+    } else if (testInfo
+        .getDisplayName()
+        .contains(
+            "test_WhenProduceRequestWithMultipleRecords_ThenRequestLogHasAggregatedErrorCodes")) {
+      restConfigs.put("dos.filter.enabled", "true");
```

Comment:
the constant for the configuration can be found in RestConfig

Reply:
I didn't see a file named `RestConfig `. Is it `KafkaRestConfig`?